internal/providers/terraform/aws/neptune_cluster_snapshot.go

```diff
@@ -0,0 +1,40 @@
+package aws
+
+import (
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+)
+
+func GetNeptuneClusterSnapshotRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "aws_neptune_cluster_snapshot",
+		RFunc: NewNeptuneClusterSnapshot,
+		ReferenceAttributes: []string{
+			"db_cluster_identifier",
+		},
+	}
+}
+
+func NewNeptuneClusterSnapshot(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	var resourceData *schema.ResourceData
+	dbClusterIdentifier := d.References("db_cluster_identifier")
+	resourceData = dbClusterIdentifier[0]
```

Comment:
it's more robust to check `if len(dbClusterIdentifier) > 0` before this line, since the reference might be in another module or project, which means we can't access it. If `db_cluster_identifier` isn't available then we should show cost-depends-on-usage - plz also add a test case for this (search the code for `account-in-another-module` to see an example of a test case that checks for this, and notice how the `account_name` is a string that doesn't exist in that TF file).